### ISSUE TITLE
Whether the memory usage of the gpu is too large

### ISSUE THREAD

**Issue by 834799106** [lucidrains/imagen-pytorch#182]
I am trying to use the combination of base64unet+SRUnet256, and max_batch_size = 4 is read from the dataset after the text embedding has generated in advance. The problem is that the above combination has occupied 45GB of GPU memory (32 precision). This makes it difficult for me to continue to expand.

**Comment by xiankgx** [comment]
Perhaps it is best to train 1 model at a time.

**Comment by 834799106** [comment]
> 也许最好一次训练 1 个模型。

Yes, I only train one model at a time, but I need two or more when I build it, otherwise I will have some problems with the first load weight when I build two models after I train one.

**Comment by deepglugs** [comment]
What are your dim_mults set at?  Also your dim settings?  I train unet2+ at 1/4 size with 1/4 random crops.  Trains with far less memory and is faster too.
